### PR TITLE
Add Building POS Channels Tutorial

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2355](https://jira.bigcommerce.com/browse/DEVDOCS-2355)

## What changed?
* Add Building POS Channel Tutorial.

### CODE REVIEW

File: docs/api-docs/channels/channels-pos-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# Building Point-of-Sale Channels
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Creating a channel](#creating-a-channel)
+- [Specifying the platform](#specifying-the-platform)
+- [Configuring UI sections](#configuring-ui-sections)
+- [Rendering UI sections](#rendering-ui-sections)
+- [Resources](#resources)
```
Comment (ektravel) [inline docs/api-docs/channels/channels-pos-tutorial.md:10]:
```suggestion
- [Related resources](#related-resources)

```

File: docs/api-docs/channels/channels-pos-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+### Prerequisites
+* [Familiarity with channels on BigCommerce](https://developer.bigcommerce.com/api-docs/channels/guide/overview)
+* [Experience building apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
+* Experience building POS integrations
+
+## Creating a channel
+
+After receiving the [POST response](https://developer.bigcommerce.com/api-docs/apps/guide/auth#receiving-the-post-response) at the end of the [single-click app OAuth flow](https://developer.bigcommerce.com/api-docs/apps/guide/auth#receiving-the-post-response), create a channel on the merchant's store. This allows merchants to import sales from the POS system and configure settings. To create a POS channel, set `type` to `pos` in the [create a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) request.
```
Comment (ektravel) [inline docs/api-docs/channels/channels-pos-tutorial.md:23]: I don't think you need to link to the same resource twice in the one sentence. 

```suggestion
After receiving the [POST response](https://developer.bigcommerce.com/api-docs/apps/guide/auth#receiving-the-post-response) at the end of the single-click app OAuth flow, create a channel on the merchant's store. This allows merchants to import sales from the POS system and configure settings. To create a POS channel, set `type` to `pos` in the [create a channel](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel) request.

```

File: docs/api-docs/channels/channels-pos-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        },
+      ]
+    }
+  }
+}
+```
+
+[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)
+
+[View reference documentation for this request](https://developer.bigcommerce.com/api-reference/store-management/channels/channels/createchannel).
```
Comment (ektravel) [inline docs/api-docs/channels/channels-pos-tutorial.md:63]: This link takes you to the Create a Channel resource, same as on line 23. (https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel)

File: docs/api-docs/channels/channels-pos-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### Exmple overview section
+
+![Example Overview Section](https://storage.googleapis.com/bigcommerce-production-dev-center/images/channels/channels-channel-overview.png "Example Overview Section")
+
+### Example import section
+
+![Example Import Section](https://storage.googleapis.com/bigcommerce-production-dev-center/images/channels/channels-pos-import.png "Example Import Section")
+
+## Resources
```
Comment (ektravel) [inline docs/api-docs/channels/channels-pos-tutorial.md:153]:
```suggestion
## Related resources

```

Comment (ektravel) [reply docs/api-docs/channels/channels-pos-tutorial.md:153]: To keep the Related resources section consistent with other articles, consider separating the resources into Articles and Endpoints.
```suggestion
## Related resources

### Articles
* [Building Apps](https://developer.bigcommerce.com/api-docs/apps/guide/intro)
* [Building Channels](https://developer.bigcommerce.com/api-docs/channels/guide/overview)
* [Find an App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id)

### Endpoints
* [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api)
* [Settings API](https://developer.bigcommerce.com/api-reference/store-management/settings)

```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**